data_structures/linked_list/linked_list.py

```diff
@@ -60,4 +54,13 @@ def insert_after(self, value, new_value):
         current = self.head
         while current != value:
             current._next = Node(new_value, current.next)
-                
\ No newline at end of file
+
+    def ll_kth_from_end(self, value):
+        current = self.head
+        if len(self) - 1 < value or value < 0:
+            print('out of range')
+            return False
+        else:
+            for _ in range(value - 1):
```

Comment:
This does not give me values as expected this gives me from the head and only works for up to and not including the last value.